MAUI/Shimmer/Customization.md

```diff
@@ -0,0 +1,227 @@
+         Text="Content is loaded!!">
+      </Label>
+   </StackLayout>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      WaveColor = Color.FromRgba("#9A4EAE"),
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## WaveWidth
+
+The width of the wave in the `SfShimmer` view can be customized using the `WaveWidth` property. The default value of `WaveWidth` is `200`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="FillAndExpand" Type="Article" Fill="Pink" WaveColor="#9A4EAE"  WaveWidth="100">
+   <StackLayout>
+      <Label 
+         Text="Content is loaded!!">
+      </Label>
+   </StackLayout>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      Fill = Colors.Pink,
+      WaveColor = Color.FromRgba("#9A4EAE"),
+      WaveWidth = 100,
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## WaveDirection
+
+The direction of the wave in the `SfShimmer` view can be customized by using the `WaveDirection` property. By default, wave direction is rendered by the `Default` wave. The following wave directions are available in SfShimmer:
+
+* `Default` - Default enum of the animation direction allows the animation from left top to right bottom.
+* `LeftToRight` - LeftToRight enum of the animation direction allows the animation from left to right.
+* `RightToLeft` - RightToLeft enum of the animation direction allows the animation from right to left.
+* `TopToBottom` - TopToBottom enum of the animation direction allows the animation from top to bottom.
+* `BottomToTop` - BottomToTop enum of the animation direction allows the animation from bottom to top.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="FillAndExpand" Type="Article" WaveDirection="TopToBottom">
+   <StackLayout>
+      <Label 
+         Text="Content is loaded!!">
+      </Label>
+   </StackLayout>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      WaveDirection = ShimmerWaveDirection.TopToBottom,
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## RepeatCount
+
+The `RepeatCount` of the shimmer refers to the number of times a shimmer cycle repeats within a given wave period in the `SfShimmer` view. The default value of `RepeatCount` is `1`.
```

Comment:
Definition is wrong

Repeat count is used to defines the number of the times the shimmer shape needed to render

Reply:
The repeat count definition is modified